src/axis.ts

```diff
@@ -44,10 +44,12 @@ export interface AxisConfig extends VgAxisConfigNoSignals, VlOnlyGuideConfig {
    *
    * __Default value:__ `true` for non-nominal fields with non-log scales; `"greedy"` for log scales; otherwise `false`.
    */
-  labelOverlap?: boolean | 'parity' | 'greedy';
+  labelOverlap?: LabelOverlap;
```

Comment:
Please check if this leads to weird "LabelOverlap" in the documentation table.  If so, we better keep `boolean | 'parity' | 'greedy'`.